activerecord/lib/active_record/associations/collection_proxy.rb

```diff
@@ -54,63 +54,6 @@ def loaded?
         @association.loaded?
       end
 
-      # Works in two ways.
```

Comment:
We should keep the documentation even that we are deleting the implementation. I think using `cal-seq directive we can keep the documentation and remove the implementation.